stuff/cron/update_ranks.py

```diff
@@ -243,6 +245,133 @@ def update_school_rank(cur: MySQLdb.cursors.BaseCursor) -> None:
                     (row['score'], rank, row['school_id']))
 
 
+def update_school_of_the_month_candidates(
+        cur: MySQLdb.cursors.BaseCursor) -> None:
+    '''Updates the list of candidates to school of the current month'''
+
+    logging.info('Updating the candidates to school of the month...')
+
+    today = datetime.date.today()
+    first_day_of_current_month = today.replace(day=1)
+    first_day_of_next_month = (
+        first_day_of_current_month + relativedelta(months=1))
+
+    # First make sure there are not already selected schools of the month
+    cur.execute(
+        '''
+        SELECT
+            *
```

Comment:
```suggestion
            COUNT(*)
```

como no se están regresando físicamente los registros y únicamente nos interesa saber si existen o no, hay que mejor usar `COUNT(*)` para que MySQL sepa que sólo queremos ver la existencia/ausencia. eso hace muchísimo más rápido todo.